volatility/framework/plugins/windows/hashdump.py

```diff
@@ -0,0 +1,345 @@
+# This file is Copyright 2020 Volatility Foundation and licensed under the Volatility Software License 1.0
```

Comment:
Hash dump's already been merged, suggesting these changes need to be rebased.  This patch won't get accepted until this hashdump is either removed, or a diff from the one in master.

The previous Hashdump required a bit of work afterwards, because the plugin wasn't actually working straight off the bat, at least not with python 3.7, and it had a number of python2 sting/byte issues which needed resolving.